packages/graph/hash_graph/lib/graph/src/types/schema/mod.rs

```diff
@@ -17,15 +17,15 @@ use core::fmt;
 
 #[doc(inline)]
 pub use self::{data_type::DataType, entity_type::EntityType, property_type::PropertyType};
-use crate::types::Uri;
+use crate::types::{BaseId, VersionedUri};
 
 #[derive(Debug)]
 pub enum ValidationError {
-    /// A schema has marked a property with a [`Uri`] as required but the [`Uri`] does not exist in
-    /// the `properties`.
-    MissingRequiredProperty(Uri),
+    /// A schema has marked a property with a [`VersionedUri`] as required but the [`VersionedUri`]
```

Comment:
The `VersionedUri`s in this doc comment should refer to `BaseId`s instead, right?